plugins/package/gxswitchlesswah/gxswitchlesswah.mk

```diff
@@ -20,7 +20,6 @@ endef
 
 define GXSWITCHLESSWAH_INSTALL_TARGET_CMDS
 	$(GXSWITCHLESSWAH_TARGET_MAKE) install DESTDIR=$(TARGET_DIR) INSTALL_DIR=/usr/lib/lv2
-	cp -rL $($(PKG)_PKGDIR)/GxSwitchlessWah.lv2/* $(TARGET_DIR)/usr/lib/lv2/GxSwitchlessWah.lv2/
```

Comment:
why did you remove this line? does your repo contain the custom ttl and modgui now?